clippy_lints/src/lib.rs

```diff
@@ -1045,6 +1046,12 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     });
     let stack_size_threshold = conf.stack_size_threshold;
     store.register_late_pass(move |_| Box::new(large_stack_frames::LargeStackFrames::new(stack_size_threshold)));
+    store.register_late_pass(move |_| {
+        Box::new(single_call_fn::SingleCallFn {
+            avoid_breaking_exported_api,
```

Comment:
Can we add this lint to the config file comment?
https://github.com/rust-lang/rust-clippy/blob/cda13a8b26d2d739fe6a4541f5773c8f2f1b3a42/clippy_lints/src/utils/conf.rs#L292-L295